docs/UserGuide.md

```diff
@@ -58,15 +60,15 @@ Athena is perfect for you if you:
 
 1. Ensure you have Java `11` or above installed in your computer.
 
-1. Download the latest `Athena.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-4/tp/releases).
+2. Download the latest `Athena.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-4/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for Athena.
+3. Copy the JAR file to the folder you want to use as the _home folder_ for Athena.
 
-1. Double-click the file or run it with `java -jar Athena.jar` to start the app. A window similar to the one below
+4. Double-click the file or run it with `java -jar Athena.jar` on Terminal to start the app. A window similar to the one below
```

Comment:
I don't think this is quite right. More specifically, the term used here should be Command Line Interface.